WordPress/openverse
# Fix flaky VCollectionHeader snapshot tests

## Fixes



Fixes https://github.com/WordPress/openverse/issues/4558 by @sarayourfriend 

## Description


Copied from the issue, where I discovered the underlying problem:

> I have a hunch, just hit me that the change is small enough that it could be flying under the default tolerance we've set. I'll try lowering it for those tests and see. This would especially make sense because only the smallest breakpoint is failing, where the text takes up a larger percentage of the screen!

That turned out to be it, exactly. I manually updated these ones by setting `maxPixelDiffRatio` to 0 for the VCollectionHeader tests. I will open an issue after finishing this PR description to dig into smarter ways of configuring `maxPixelDiffRatio` that are aware of things like our varying screen sizes and the effect that might have on things.

It might mainly affect only the storybook visual regression tests because they are so much smaller overall? Anyway, something to think about in the other issue, not here. For now, I just want to fix the flaky test by updating these snapshots to have the correct text.

## Testing Instructions

CI must pass. Confirm the updated screenshots use the correct text and look right otherwise.

## Checklist


- [N/A] I added or updated documentation (if applicable).
- [N/A] I ran the DAG documentation generator (`./ov just catalog/generate-docs` for catalog
      PRs) or the media properties generator (`./ov just catalog/generate-docs media-props`
      for the catalog or `./ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
